.github/workflows/test.yml

```diff
@@ -286,6 +286,7 @@ jobs:
 
     - name: Free up disk space
       run: |
+        sudo rm -rf /usr/local/lib/android/sdk
```

Comment:
8gb.. ? really?
what was this pkg being used for?

Reply:
@jcaamano found it, I suppose is cooked by github at the ubuntu image, so they don't spend time installing stuff.
```
8.9G    /usr/local/lib/android/sdk
```